libsolidity/interface/CompilerStack.cpp

```diff
@@ -1013,15 +1016,34 @@ Json::Value const& CompilerStack::natspecDev(Contract const& _contract) const
 	return _contract.devDocumentation.init([&]{ return Natspec::devDocumentation(*_contract.contract); });
 }
 
-Json::Value CompilerStack::methodIdentifiers(string const& _contractName) const
+Json::Value CompilerStack::contractIdentifiers(string const& _contractName) const
```

Comment:
Can we choose a less generic name for this method? It does not return all identifiers of the contract...

Reply:
Well or I guess `interfaceIdentifiers`?

Reply:
`interfaceMethodsErrorsAndEvents`?